skins/10h16/templates/payment_preset_amounts.html.twig

```diff
@@ -0,0 +1,8 @@
+{% for amount in presetAmounts if amount.getEuroCents() != 30000 %}
+    <li>
+        <input type="radio" class="amount-select" id="form{$ formId $}-amount-{$ loop.index $}" name="betrag_auswahl" value="{$ amount.getEuroFloat() | number_format( 2, ',', '' ) $}" title="{$ amount.getEuroFloat() | number_format( 0 ) $}&euro;">
```

Comment:
Not a blocker, but for the sake of consistency the value access could be aligned with how it's done in cat17 now.